libs/vkd3d/device.c

```diff
@@ -2933,7 +2933,10 @@ static HRESULT STDMETHODCALLTYPE d3d12_device_CheckFeatureSupport(d3d12_device_i
             if (image_features & VK_FORMAT_FEATURE_BLIT_SRC_BIT)
                 data->Support1 |= D3D12_FORMAT_SUPPORT1_MULTISAMPLE_RESOLVE;
             if (image_features & VK_FORMAT_FEATURE_STORAGE_IMAGE_BIT)
+            {
                 data->Support1 |= D3D12_FORMAT_SUPPORT1_TYPED_UNORDERED_ACCESS_VIEW;
+                data->Support2 |= D3D12_FORMAT_SUPPORT2_UAV_TYPED_LOAD | D3D12_FORMAT_SUPPORT2_UAV_TYPED_STORE;
```

Comment:
This needs to check if ReadWithoutFormat and WriteWithoutFormat is supported I think.

Reply:
Is it because` Load/Store `in HLSL does not have equivalent `OpTypeImage` ?

Reply:
Yes, all typed UAV loads and stores in DX are without format in the shader. The exception to this rule is FL 11.0 requirements where typed loads only work on R32_UINT/SINT/FLOAT, so we can infer the format in certain scenarios. Typed stores are without format always.

Reply:
Thanks for explaining.!